internal/provider/aws.go

```diff
@@ -87,3 +90,29 @@ func (p *AWSTrialInputProvider) region() string {
 	}
 	return DefaultAWSTrialRegion
 }
+
+func (p *AWSFreemiumInputProvider) Provide() Values {
+	machineType := DefaultOldAWSTrialMachineType
+	if p.UseSmallerMachineTypes {
+		machineType = DefaultAWSMachineType
+	}
+	region := p.region()
+
+	return Values{
```

Comment:
There is a lot of inputs set here: https://github.com/kyma-project/kyma-environment-broker/blob/b4a26925fa1ae24f1f063e6d12e92146b6aed064/internal/provider/azure_provider.go#L197-L198. Is that the correct place that should server as a template of what to sent to RunimeResource?